geode-core/src/integrationTest/java/org/apache/geode/cache/query/internal/index/IndexUseJUnitTest.java

```diff
@@ -353,7 +353,7 @@ public void testCompactMapIndexUsageWithIndexOnSingleKey() throws Exception {
 
     Index i2 =
         qs.createIndex("Index2", IndexType.FUNCTIONAL, "objs.liist[0]", SEPARATOR + "testRgn objs");
-    assertTrue(i2 instanceof CompactRangeIndex);
+    assertTrue(i2 instanceof CompactMapRangeIndex);
```

Comment:
I highly recommend updating any test that you touch to use AssertJ instead of JUnit Assert. You can even install an IntelliJ plugin that'll automate most of the work for you.

JUnit assertions like this will produce a terrible failure message:
```
ava.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at org.apache.geode.IndexStatisticsJUnitTest.testCompactMapIndexUsageWithIndexOnSingleKey(IndexStatisticsJUnitTest.java:356)
```
Changing to AssertJ:
```
assertThat(i2).isInstanceOf(CompactMapRangeIndex.class);
```
The failure message will then be:
```
java.lang.AssertionError: 
Expecting:
  java.lang.Object@735b478
to be an instance of:
  java.util.Collection
but was instance of:
  java.lang.Object

	at org.apache.geode.IndexStatisticsJUnitTest.testCompactMapIndexUsageWithIndexOnSingleKey(IndexStatisticsJUnitTest.java:356)
```